src/main/docs/guide/httpServer/jsonBinding.adoc

```diff
@@ -153,3 +153,14 @@ jackson:
     useBigDecimalForFloats: true
 ----
 
+== Text Binding
+
+Micronaut Framework supports "text/plain" Content-Type as a way of binding to plain text. However, `text/plain` messages are more restrictive about allowed types. To return plain text responses for answers other than strings, manually call the object `toString()` method.
```

Comment:
more restrictive than what?

Reply:
Maybe you can tell me. I copied what you wrote. 😉

https://github.com/micronaut-projects/micronaut-core/commit/37874c634202233f35b7c9376a5edfd5d49861f2#diff-2544d80c201711826bfd793aeda3dfec281d52b26c0bfa75ee20d019fc55f229:~:text=%3D%3D%3D%3D%20%60text/plain%60%20messages,g.%20number%20types).

Reply:
I'll add that phrase for clarity. Thanks.

Reply:
@yawkat I clarified the docs a little bit.